resources.storage.account.private.endpoints.table.tf

```diff
@@ -10,18 +10,25 @@ data "azurerm_virtual_network" "table_vnet" {
   resource_group_name = local.resource_group_name
 }
 
+data "azurerm_subnet" "snet" {
```

Comment:
`terraform init` is throwing an error on this data confituration in a clean envoironment.

` Error: Duplicate data "azurerm_subnet" configuration
│
│   on ../../resources.storage.account.private.endpoints.table.tf line 13:
│   13: data "azurerm_subnet" "snet" {
│
│ A azurerm_subnet data resource named "snet" was already declared at
│ ../../resources.storage.account.private.endpoints.blob.tf:13,1-29. Resource names must be
│ unique per type in each module.`

Reply:
Ok added naming fixes:

data.azurerm_virtual_network.blob_snet.0.name & data.azurerm_virtual_network.table_snet.0.name

Also fixed the VNET naming as it needed data. in front of it